src/widget/woverview.cpp

```diff
@@ -699,6 +700,42 @@ void WOverview::drawWaveformPixmap(QPainter* pPainter) {
     }
 }
 
+void WOverview::drawMinuteMarkers(QPainter* pPainter) {
+    if (!m_trackLoaded)
+        return;
+    WaveformWidgetFactory* widgetFactory = WaveformWidgetFactory::instance();
+    if (!widgetFactory->isOverviewMinuteMarkers())
+        return;
+
+    // Faster than track->getDuration() and already has playback speed ratio compensated for
+    double trackSeconds = samplePositionToSeconds(getTrackSamples());
+
+    QLineF line;
+    pPainter->setPen(QPen(m_axesColor, m_scaleFactor));
+    pPainter->setOpacity(1);
+
+    double overviewHeight = height();
+    double markerHeight = overviewHeight * 0.08;
+    double lowerMarkerYPos = overviewHeight * 0.92;
+    double currentMarkerXPos;
+    double currentMarkerSeconds = 60;
+    while (1) {
```

Comment:
```suggestion
    const double overviewHeight = height();
    const double markerHeight = overviewHeight * 0.08;
    const double lowerMarkerYPos = overviewHeight * 0.92;
    const double currentMarkerXPos;
    for (double currentMarkerSeconds = 60; currentMarkerSeconds <= trackSeconds; currentMarkerSeconds += 60) {
```

Reply:
we don't need a marker at the end if that is 60*N, so it would be
`for (double currentMarkerSeconds = 60; currentMarkerSeconds < trackSeconds; currentMarkerSeconds += 60) {`

Reply:
fixed